cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -150,8 +155,21 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
             sourceControlUsernameCredentials.name,
             mavenCredentialsUsername.name
         );
+        binaryApplication.perform_assessment("low", [stakeHolder]);
         binaryApplication.analyze();
         binaryApplication.verifyAnalysisStatus("Completed");
         binaryApplication.selectApplication();
     });
+
+    it("Assess application", function () {
+        const assessment = new Analysis(
+            getRandomApplicationData("tackletestApp_binary", {
+                binaryData: this.appData["tackle-testapp-binary"],
+            }),
+            getRandomAnalysisData(this.analysisData["binary_analysis_on_tackletestapp"])
+        );
+        assessment.name = this.upgradeData.assessmentName;
```

Comment:
application.name = this.upgradeData.assessmentName;

Reply:
@sshveta this is just a variable name, I named it this way to make a difference. In any way, this is instance of `Analysis` class